include/seqan3/utility/bloom_filter/bloom_filter.hpp

```diff
@@ -0,0 +1,385 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Tobias Loka <tobias.loka AT hpi.de>
+ * \brief Provides seqan3::bloom_filter.
+ */
+
+#pragma once
+
+#include <seqan3/std/algorithm>
+#include <seqan3/std/bit>
+#include <seqan3/search/dream_index/interleaved_bloom_filter.hpp>
+
+#include <sdsl/bit_vectors.hpp>
+
+#include <seqan3/core/concept/cereal.hpp>
+#include <seqan3/core/detail/strong_type.hpp>
```

Comment:
```suggestion
#pragma once

#include <seqan3/search/dream_index/interleaved_bloom_filter.hpp>
```

This is enough for now.

Since it is `utility`, we would need to decouple this from anything that is not `seqan3/core`, i.e. we would need to move the `data_layout` from the `interleaved_bloom_filter.hpp` to .... `core`?

But this can be done later.